apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/MonitoredResourceUtils.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
This class (or something similar to it) should probably go in Google.Apis.Logging.v2
Then this code and the log4net code can share it.